app/models/public_body.rb

```diff
@@ -202,6 +202,14 @@ def set_api_key!
     self.api_key = SecureRandom.base64(33)
   end
 
+  def self.find_by_name(name)
+    find_by(name: name)
```

Comment:
Not sure if we need this method – doesn't appear to be called anywhere in `app/`